content/en/tracing/setup_overview/compatibility_requirements/dotnet-framework.md

```diff
@@ -15,51 +15,75 @@ further_reading:
+| ------------------------------- | ----------------------------------------------------------------------------------------- | -------------------- |
+| ADO.NET                         | built-in                                                                                  | `AdoNet`             |
+| ASP.NET (including Web Forms)   | built-in                                                                                  | `AspNet`             |
+| ASP.NET MVC                     | `Microsoft.AspNet.Mvc` 4.0+                                                               | `AspNetMvc`          |
+| ASP.NET Web API 2               | `Microsoft.AspNet.WebApi` 5.1+                                                            | `AspNetWebApi2`      |
+| AWS SQS                         | `AWSSDK.SQS`  3.0+                                                                        | `AwsSqs`             |
+| CosmosDb                        | `Microsoft.Azure.Cosmos.Client` 3.6.0                                                     | `CosmosDb`           |
+| Elasticsearch                   | `Elasticsearch.Net` 5.3.0+                                                                | `ElasticsearchNet`   |
+| HttpClient / HttpMessageHandler | built-in                                                                                  | `HttpMessageHandler` |
+| Kafka                           | `Confluent.Kafka` 1.4+                                                                    | `Kafka`              |
+| MongoDB                         | `MongoDB.Driver.Core` 2.1.0+                                                              | `MongoDb`            |
+| MSMQ                            | built-in                                                                                  | `Msmq`               |
+| MySql                           | `MySql.Data` 6.7.0+                                                                       | `AdoNet`             |
+| Oracle                          | `Oracle.ManagedDataAccess` 4.122.0+                                                       | `AdoNet`             |
+| PostgreSQL                      | `Npgsql` 4.0+                                                                             | `AdoNet`             |
+| RabbitMQ                        | `RabbitMQ.Client` 3.6.9+                                                                  | `RabbitMQ`           |
+| Redis (ServiceStack client)     | `ServiceStack.Redis` 4.0.48+                                                              | `ServiceStackRedis`  |
+| Redis (StackExchange client)    | `StackExchange.Redis` 1.0.187+                                                            | `StackExchangeRedis` |
+| SQL Server                      | `System.Data` 4.0.0+, `System.Data.SqlClient` 4.0.0 +, `Microsoft.Data.SqlClient` 1.0.0+  | `AdoNet`             |
```

Comment:
We want to force line breaks here. See [this comment](https://github.com/DataDog/documentation/pull/11200/files#r676861334).

```suggestion
| SQL Server                      | `System.Data` 4.0.0+</br>`System.Data.SqlClient` 4.0.0+</br>`Microsoft.Data.SqlClient` 1.0.0+  | `AdoNet`             |
```